notebooks/official/feature_store/vertex_ai_feature_store_feature_view_service_agents.ipynb

```diff
@@ -0,0 +1,840 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    PROJECT_ID = ""  # @param {type:"string"}
</pre><p>Use PROJECT_ID = "[your-project-id]"  <em># @param {type:"string"}</em></p><p> </p><p>[your-project-id] will be automatically replaced by the product ID used  in the test framework.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2752/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>